pipelines/h6c/idr2/v2/lstbin/lstbin-template.toml

```diff
@@ -0,0 +1,76 @@
+#! This is a *template* file for the lstbin makeflow. Use `herapipes lstbin-setup` to 
```

Comment:
Do we need to modify anything in h6c/idr2/v2 when we're moving on to v3?

Reply:
no. I believe I was updating this to use on v2, but then we decided to move to v3 instead. I can remove the v2 files.

Reply:
sounds good